pkg/ucp/api/v20231001preview/awsplane_conversion_test.go

```diff
@@ -41,7 +41,7 @@ func Test_AWSPlane_ConvertVersionedToDataModel(t *testing.T) {
 					TrackedResource: v1.TrackedResource{
 						ID:       "/planes/aws/aws",
 						Name:     "aws",
-						Type:     "System.AWS/planes",
+						Type:     datamodel.AWSPlaneResourceType,
```

Comment:
While working on this feature I noticed that we didn't have a single definition for these values, so I added one. This is just a refactor and won't affect behaviors.